Sources/TarkaUI/Components/Search Bar/SearchTextField.swift

```diff
@@ -15,7 +15,8 @@ struct SearchTextField: View {
   
   var body: some View {
     
-    TextField(searchBarVM.searchItem.placeholder, text: $searchBarVM.searchItem.text)
+    TextField("", text: $searchBarVM.searchItem.text,
+              prompt: Text(searchBarVM.searchItem.placeholder).foregroundColor(.inputTextDim))
```

Comment:
```suggestion
              prompt: Text(searchBarVM.searchItem.placeholder)
              .foregroundStyle(Color.inputTextDim))
```
Better use `foregroundStyle`

Reply:
that could be iOS 17 and later only